services/galley/test/integration/API.hs

```diff
@@ -1906,7 +1920,7 @@ testGetQualifiedRemoteConv = do
   conv <- responseJsonUnsafe <$> (pure respAll <!! const 200 === statusCode)
   -- FUTUREWORK: The backend should augment returned conversation data with
   -- Alice's membership data stored locally
```

Comment:
Should this futurework note go?

Reply:
Yes, done in fe55625f3a10ecd600b47e35cf90042df9403857.